shift_dev/vis/video.py

```diff
@@ -150,7 +150,7 @@ def main():
         args.view,
         vis_type,
         video_path,
-        convert_to_gray=args.gray_background,
+        convert_to_gray=args.no_gray_background,
```

Comment:
Thanks for the fix! Could you please update this as `not args.no_gray_background`?